pkg/controller/configmap_ca_injector/controller.go

```diff
@@ -5,7 +5,8 @@ import (
 	"fmt"
 	"log"
 
-	oclient "github.com/openshift/cluster-network-operator/pkg/client"
+	"github.com/openshift/cluster-network-operator/pkg/apply"
+	client "github.com/openshift/cluster-network-operator/pkg/client"
```

Comment:
Not a fan of the name 'client'. Rename to cnoclient?

Reply:
done.